openshift/templates/angular-on-nginx/angular-on-nginx.bc.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: ${NAME}
+objects:
+  - kind: ImageStream
+    apiVersion: image.openshift.io/v1
+    metadata:
+      labels:
+        app: ${GROUP_NAME}
+      name: ${NAME}
+  - apiVersion: build.openshift.io/v1
+    kind: BuildConfig
+    metadata:
+      labels:
+        app: ${GROUP_NAME}
+      name: ${NAME}-build
+    spec:
+      completionDeadlineSeconds: 1800
+      failedBuildsHistoryLimit: 5
+      nodeSelector: null
+      output:
+        to:
+          kind: ImageStreamTag
+          name: ${NAME}:latest
+      postCommit: {}
+      resources:
+        limits:
+          cpu: "2"
+          memory: 2Gi
+        requests:
+          cpu: "1"
+          memory: 1Gi
+      runPolicy: Parallel
```

Comment:
I think we'll want Serial here

Reply:
https://github.com/bcgov/mem-mmti-public/pull/344/commits/33e3d5b17e35d174f683cf035d6872846e50195c